aws-route53/README.md

```diff
@@ -38,7 +38,7 @@ SignalFx provides built-in dashboards for this service. Examples are shown below
 
 ### METRICS
 
-For more information about the metrics emitted by Amazon Route 53, visit the service's homepage at https://aws.amazon.com/route53/.
+For more information about the metrics emitted by Amazon Route 53, visit the service's homepage at <a target="_blank" href="https://aws.amazon.com/route53/">https://aws.amazon.com/route53/</a>.
```

Comment:
This is kind of nit picky but can't we just say `visit <a href="..." target="_blank">the service's homepage</a>.` instead?

Same thing applies to a bunch of the other AWS integration docs.

Reply:
Yes, we can. But, I did not want to change the content, just the URL syntax. I think the content should be edited in a separate PR rather than part of this behemoth.